crates/router/src/compatibility/stripe/payment_intents/types.rs

```diff
@@ -669,6 +744,15 @@ impl ForeignFrom<Option<Request3DS>> for api_models::enums::AuthenticationType {
     }
 }
 
+impl From<AcceptanceType> for payments::AcceptanceType {
```

Comment:
I guess this impl is already there. If yes please use existing one.

Reply:
Okay, I'll check